src/dev/konnector.md

```diff
@@ -508,6 +508,33 @@ Cozy will soon have a store for connectors and you will be able to publish conne
 at the moment, you need to declare your new connector on the [cozy forum](https://forum.cozy.io).
 The Cozy team will review your code and add your connector to the [Cozy Collect] application.
 
+To make the connector available more quickly for all cozys, you can follow this few steps of 
+pre-packaging:
+
+### Icon
+
+You need to push an icon in `assets/icon.png`. Please respect this rules :
+ - Square icon, possibly a png or svg
+ - Try the apple store icon if needed
+ - Set the correct name in manifest
+
+### Package.json
+
+ - Edit the name to be cozy-konnector-slug
+ - Edit the repository
+ - Edit the command `deploy` with the correct repository
+
+### Manifest.konnector
+
+ - Edit the name with a nice name (Capitals and spaces allowed here)
+ - Edit icon as needed
+ - Edit slug
+ - Edit source with the correct repository
+ - Add a correct vendor link
+ -  Choose one or more categories in this list (list in collect) : `banking, shopping, insurance, isp, telecom, energy, public_service, other`
```

Comment:
there is an extra space after the dash

Reply:
Don't duplicate categories names here. It creates two sources of truth and as soon as sources of truth will be desynchronized, contributors may be confused. Use a url to link to the source of truth.